### PR TITLE
#6708 - how to use images in code

### PR DESCRIPTION
**Pull request by thnaylor**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes [#6708](https://github.com/kodadot/nft-gallery/issues/6708)
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=123PCJXhjb15i6JwVVRGd7KvN2sSNZrtPjr5doJq9oWctoTt)

## Screenshot
- Added lint warning when importing common image file types in page, layouts & component directories.
image

- Updated STYLE_GUIDE.md with new conventions as mentioned in #6707 & #6708
image
image

### <samp>🤖 Generated by Copilot at 50d178b</samp>

This pull request enhances the style guide for image handling and adds a linting rule to enforce it. It aims to improve the performance and accessibility of the NFT gallery website by using HTML tags and URL paths for images instead of JavaScript imports.

### <samp>🤖 Generated by Copilot at 50d178b</samp>

> _`ESLint` warns us_
> _Images in JavaScript_
> _Harm the web in spring_

### CODE REVIEW

**Comment by yangwao** [comment]
exactly 

image

File: .eslintrc.js [hunk 1]
```diff
@@ -45,7 +45,33 @@ module.exports = {
...
     {
-      files: ['layouts/**/*.{js,ts,vue}', 'pages/**/*.vue'],
+      files: [
+        'layouts/**/*.{js,ts,vue}',
+        'pages/**/*.vue',
+        'components/**/*.{js,ts,vue}',
+      ],
+      rules: {
+        'no-restricted-imports': [
+          'warn',
```
Comment (preschian) [inline .eslintrc.js:55]: Cool. Can we force it as an 'error'? Haha 🤣

**Review by preschian — APPROVED** [review]

File: .eslintrc.js [hunk 2]
```diff
@@ -45,7 +45,33 @@ module.exports = {
   ignorePatterns: ['*.md'],
   overrides: [
     {
-      files: ['layouts/**/*.{js,ts,vue}', 'pages/**/*.vue'],
+      files: [
+        'layouts/**/*.{js,ts,vue}',
+        'pages/**/*.vue',
+        'components/**/*.{js,ts,vue}',
```
Comment (preschian) [inline .eslintrc.js:51]: oh, need more folders, just in case

`utils`, `stores`, `services`, `plugins`, `composables`

File: .eslintrc.js (hunk 1 above)
Comment (thnaylor) [reply .eslintrc.js]: Thanks, I have updated the rule to an 'error'.

The linter will now  'ELIFECYCLE  Command failed with exit code 1".

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 4.75 USD/DOT ~ 10.526 $DOT
🧗 [123PCJXhjb15i6JwVVRGd7KvN2sSNZrtPjr5doJq9oWctoTt](https://polkadot.subscan.io/account/123PCJXhjb15i6JwVVRGd7KvN2sSNZrtPjr5doJq9oWctoTt)
🔗 [0x8483e01ccbf236e7ca4b27308fa3e4fcfe4603d45cbaf76e19d43950cfbb28a2](https://polkadot.subscan.io/extrinsic/0x8483e01ccbf236e7ca4b27308fa3e4fcfe4603d45cbaf76e19d43950cfbb28a2)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**